lib/web_ids.js

```diff
@@ -46,39 +42,85 @@ function init() {
  */
 function loadSignatures(dirname) {
   fs.readdirSync(dirname).forEach((filename) => {
-    content = JSON.parse(fs.readFileSync(dirname + filename));
-    content["ipSignature"].forEach((x) => ipSignature.push(x));
-    content["headerSignature"].forEach((x) => headerSignature.push(x));
-    content["bodySignature"].forEach((x) => bodySignature.push(x));
-    content["statusSignature"].forEach((x) => statusSignature.push(x));
+    const filePath = dirname + '/' + filename;
+    let content = null;
+
+    try {
+      content = JSON.parse(fs.readFileSync(filePath));
+    } catch (syntaxError) {
+      console.error('\n' + filePath + ' is not a valid JSON. Skipping file.\n');
```

Comment:
I think it is better to only read files that are json files. so checking the mimetype maybe before parsing?

Reply:
That is correct, will be part of the next commit. However, file can be JSON and still have wrong syntax, so I will leave this code also there.